.travis.yml

```diff
@@ -17,14 +17,7 @@
 dist: trusty
 sudo: required
 addons:
-  chrome: stable
-
-env:
-  - SCRIPT="mvn surefire:test@unit-tests -T 2C"
-  - SCRIPT="mvn surefire:test@integration-tests"
-  - SCRIPT="mvn clean site --projects site-book"
-  - SCRIPT="mvn test --projects metron-interface/metron-config,metron-interface/metron-alerts"
-  - SCRIPT="./dev-utilities/build-utils/verify_licenses.sh"
+ chrome: stable
```

Comment:
Is the space removal intentional? If so, is there an intended side effect to that? The other indents are 2 spaces.

Reply:
Not intentional.  I'll pull that back.